spec/rubocop/cop/lint/deprecated_gem_version_spec.rb

```diff
@@ -3,7 +3,8 @@
 
 RSpec.describe RuboCop::Cop::Lint::DeprecatedGemVersion do
   subject(:cop) { described_class.new(config) }
-  let(:config) { RuboCop::Config.new }
+  let(:empty_rubocop_config) { { } }
+  let(:config) { RuboCop::Config.new(empty_rubocop_config) }
```

Comment:
This was the default value in Rubocop before this commit: https://github.com/rubocop/rubocop/commit/7e73f51a2eca280fae2780d3d6190ebdb1d150d1#diff-b35fca184e57be3f44600ddadf6c2ceee757410050664099239e990a903d85a9

Now we explicitly pass in `{}` to avoid reading the default rubocop config